nvtabular/workflow/node.py

```diff
@@ -279,10 +391,18 @@ def label(self):
 
     @property
     def _cols_repr(self):
-        cols = ", ".join(map(str, self.selector[:3]))
-        if len(self.selector) > 3:
-            cols += "..."
-        return cols
+        if self.input_schema:
```

Comment:
This fixes the string representation of the columns, but there's also an issue where nodes that are only dependencies and not parents think they're output nodes (according to the string representation.) We might be able to get around that by making the nodes that depend on them children?